### PR TITLE
React capstone project

### PR DESCRIPTION
**Pull request by kessio**
Hello 👋🏾 

**### kindly: my linters is claiming lock-JSON file has been created for version 2 instead of 1. Kindly how do I solve that, I've tried everything?**

This PR contains a crypto web application that displays the top 100 cryptocurrencies in the market

The following objectives have been implemented:
Use React documentation.
Use React components.
Use React props.
Use React Router.
Connect React and Redux.
Handle events in a React app.
Write integration tests with a React testing library.
Use styles in a React app.
Use React life cycle methods.
Apply React best practices and language style guides in code.
Use store, actions, and reducers in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,77 @@
-# crypto-price-tracker
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```
Comment (SajjadAhmad14) [inline README.md]: Please add `project name` and give appropriate credit to the design author in the `Acknowledgements` section of README.

File: src/components/Details.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import React, { useEffect } from 'react';
```
Comment (SajjadAhmad14) [inline src/components/Details.js:1]: Please add a `Back` button on the Detail page on the Desktop version that should take the user back to the Home page.

File: README.md [hunk 3]
```diff
@@ -1 +1,77 @@
...
+
+## Built With
+
+- Major languages : Html5, Javascript, CSS
+- Frameworks: React, Redux, TailwindCSS
+- Technologies used: Git, npm, linters
+
+## Live Demo 
+
+[Live Demo Link](https://livedemo.com)
```
Comment (SajjadAhmad14) [inline README.md]: Please deploy the app on Heroku or Netlify and add a link to it in README.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Sharon, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
The project has a presentation video. :heavy_check_mark:

## To Improve 🙏
- [ ] PR has some failing linter checks. Kindly make sure that all the linter checks are green.
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by cwaku — APPROVED** [review]
Hi @kessio 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions
- [ ] I suggest that you separate the words from the numbers on the details page. Preferably use a colon and space.
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. The previous reviewer simply missed some of them, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**